src/store/metadata/stub/content/editions/pupils-at-expected-standards.json

```diff
@@ -0,0 +1,130 @@
+      "contact_point": {
+        "name": "Ofcom Open Data",
+        "email": "mailto:open.data@ofcom.org.uk"
+      },
+      "topics": [
+        "https://staging.idpd.uk/topics/children-education-and-skills"
+      ],
+      "frequency": "monthly",
+      "keywords": [
+        "reading",
+        "writing",
+        "maths",
+        "subnational",
+        "region",
+        "county",
+        "state-funded-school",
+        "key-stage-2-attainment"
+      ],
+      "licence": "http://www.nationalarchives.gov.uk/doc/open-government-licence/version/3/",
+      "issued": "2023-03-31T12:15:00Z",
+      "modified": "2023-08-23T15:55:00Z",
+      "spatial_resolution": [
+        "N09",
+        "S12",
+        "W06",
+        "E92",
+        "K02"
+      ],
+      "spatial_coverage": "K02000001",
+      "temporal_resolution": [
+        "P1M",
+        "P3M"
+      ],
+      "temporal_coverage": {
+        "start": "2022-09-01T00:00:00Z",
+        "end": "2022-09-30T23:59:00Z"
+      },
+      "versions_url": "https://staging.idpd.uk/datasets/pupils-at-expected-standards/editions/2023-03/versions",
+      "versions": [
+        {
+          "@id": "https://staging.idpd.uk/datasets/pupils-at-expected-standards/editions/2023-03/versions/1",
+          "issued": "2023-03-31T12:15:00Z"
+        }
+      ],
+      "next_release": "2024-04-19T09:30:00Z",
+      "table_schema": {
+        "columns": [
+          {
+            "name": "period",
+            "datatype": "string",
+            "titles": "Period",
+            "description": "The time period associated with the observation."
+          },
+          {
+            "name": "areacd",
+            "datatype": "string",
+            "titles": "ONS Geography Code",
+            "description": "The geography code associated with the observation."
+          },
+          {
+            "name": "areanm",
+            "datatype": "string",
+            "titles": "Area Name",
+            "description": "The name assiciated with The geography code "
+          },
+          {
+            "name": "geography",
+            "datatype": "string",
+            "titles": "Geography Type",
+            "description": "The categorisation of the geography code"
+          },
+          {
+            "name": "value",
+            "datatype": "decimal",
+            "titles": "Geographic area with percentage of pulpils at expected standards by at least one provider",
+            "description": "Observational Value"
+          },
+          {
+            "name": "measure",
+            "datatype": "string",
+            "titles": "Measure Type",
+            "description": "Measure Type of the Observational Value"
+          },
+          {
+            "name": "unit",
+            "datatype": "string",
+            "titles": "Unit",
+            "description": "Unit Type of the Observational Value"
+          },
+          {
+            "name": "lower confidence interval (95%)",
```

Comment:
These names should be consistent with the other recently updated datasets: lower_confidence_interval_95 instead of lower confidence interval (95%)

Reply:
done